discretize/TreeMesh.py

```diff
@@ -69,15 +69,22 @@
 #                   fZm
 #
 #
-#            fX                                  fY                                 fZ
-#      2___________3                       2___________3                      2___________3
-#      |     e1    |                       |     e1    |                      |     e1    |
-#      |           |                       |           |                      |           |
-#   e2 |     x     | e3      z          e2 |     x     | e3      z         e2 |     x     | e3      y
-#      |           |         ^             |           |         ^            |           |         ^
-#      |___________|         |___> y       |___________|         |___> x      |___________|         |___> x
-#      0    e0     1                       0    e0     1                      0    e0     1
-#
+#            fX                                  fY
+#      2___________3                       2___________3
+#      |     e1    |                       |     e1    |
+#      |           |                       |           |
+#   e2 |     x     | e3      z          e2 |     x     | e3      z
+#      |           |         ^             |           |         ^
+#      |___________|         |___> y       |___________|         |___> x
+#      0    e0     1                       0    e0     1
+#           fZ
+#      2___________3
+#      |     e1    |
+#      |           |
+#   e2 |     x     | e3      y
+#      |           |         ^
+#      |___________|         |___> x
+#      0    e0     1
```

Comment:
I like how you have fixed up my comments.

Reply:
PEP8 flags are hurting my OCD brain :)